apps/src/sites/studio/pages/components/FlexGroup.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+import color from '../../../../util/color';
```

Comment:
I wonder if we should start using `@cdo/apps/utils/color` instead of these long paths of `../` that are hard to follow. I think @pcardune had suggested that a good heuristic might be to provide relative paths for things below you (i.e. if your path starts with `./` but use `@cdo/apps` otherwise. Should we formalize that? (Not sure if there are downsides to using @cdo/apps in mroe places.